worldengine/World.proto

```diff
@@ -1,3 +1,4 @@
+syntax = "proto2";
```

Comment:
Here is the API/docs for proto2 and proto3.  Proto3 just extends proto2 and should be backwards compatible.